app/services/notices/setup/send-notice.service.js

```diff
@@ -92,6 +104,16 @@ async function _sendNotifications(notifications, referenceCode) {
   return sentNotifications
 }
 
+async function _updateFailedEmailInvitations(noticeId) {
+  return NotificationModel.query()
```

Comment:
It used to be that I dictated all database calls 'returned' rather than 'awaited'. But then we did a deep dive and learnt that there is no performance or coding value in doing this, and that it actually does not reflect the intent of the code.

In this case, we intend to wait until the records have been updated, and therefore, this should be `await` to reflect this.


```suggestion
  await NotificationModel.query()
```